Feature: USWDS update and remove node sass

jbrown-xentity opened the pull request:
Related to: 

- https://github.com/GSA/datagov-deploy/issues/2566
- https://github.com/GSA/datagov-deploy/issues/2807
- https://github.com/GSA/datagov-deploy/issues/2808

adborden commented on .snyk:
```diff
@@ -50,62 +38,6 @@ ignore:
-    - gulp-sass > node-sass:
-      reason: Created issue to resolve in 90-days
-      expires: '2021-05-14T21:40:40.202Z'
 # patches apply the minimum changes required to fix a vulnerability
```
You should remove these patches. I'm not seeing `snyk patch` in the package.json scripts, so these could be hiding true positives.

adborden commented on gulpfile.js:
```diff
@@ -1,14 +1,14 @@
 var autoprefixer = require("autoprefixer");
-var autoprefixerOptions = require("./node_modules/uswds-gulp/config/browsers");
-var cssnano = require("cssnano");
+// var autoprefixerOptions = require("./node_modules/uswds-gulp/config/browsers");
```
You can delete this.
```suggestion
```

adborden reviewed (approved):
👏  excellent!

jbrown-xentity commented:
I removed gulp-svg-sprite, even though it was [required by uswds-gulp](https://github.com/uswds/uswds-gulp#requirements). Site built and seems to work fine, although someone should check if they know what page a sprite might exist on...
Removed because it had dependencies that were old/deprecated, and no great remediation path available.
